### PR TITLE
tests/ci: add grafiti multibranch jenkins job

### PR DESCRIPTION
**Pull request by cpanato**
Before merging this we need to delete the existing Jenkins job
this will replace the existing Jenkins job

cc @mxinden @estroz

### CODE REVIEW

**Comment by estroz** [comment]
@cpanato has this script been tested?

**Comment by cpanato** [comment]
yes, using another name to not mess the existing one

**Comment by estroz** [comment]
sgtm. wait for @mxinden's approval to merge

**Comment by cpanato** [comment]
we just need to sync, when @mxinden approve, I can merge and do all the work :)

File: jenkins-jobs/grafiti_multibranch_job.groovy [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/env groovy​
+
+multibranchPipelineJob("grafiti") {
+  description("Tag and remove AWS Resources with Automation.\nThis job is manage by tectonic-installer.\nChanges here will be reverted automatically.")
```
Comment (mxinden) [inline jenkins-jobs/grafiti_multibranch_job.groovy]: @cpanato How is this managed by the tectonic-installer?
s/manage/managed

Comment (cpanato) [inline jenkins-jobs/grafiti_multibranch_job.groovy]: should be grafiti repo. will change

**Comment by estroz** [comment]
@cpanato @mxinden lgtm. add @ecordell to future PR reviewers too

**Pull request merged**